### ISSUE TITLE
Open source workflow in R for spatial data and analysis

### ISSUE THREAD

**Issue by jamiecmontgomery** [ropensci/unconf#29]
What are the best workflows and packages for spatial data and analyses? 

We do a lot of geospatial processing in the Ocean Health Index (see Issue #28), and we would like to streamline our workflow and to ensure all our analysis is done on open source platforms, ideally R.  We currently do some spatial analyses in R using packages such as raster, sp, maps, rgdal, and more, but we still rely heavily on Python and ArcGIS because of limitations we have encountered in R (e.g., dealing with large shapefiles, etc.).  

If this is a problem others encounter often enough, it would be great to develop an effort to do/discuss/improve either or all of the following:
- Is it possible to develop methods so spatial vector objects (i.e., shapefiles, etc.) aren’t read 
  into the working memory, similar to the way that the raster package deals with raster files? 
- Identify limitations to spatial analysis in R and identify workarounds to these issues
- Best approaches for integrating QGIS/Python/R

If this gains interest I'm happy to have this topic take a form that's most conducive to the community

**Comment by karthik** [comment]
I'd be interested in this discussion.

**Comment by karthik** [comment]
This went on CRAN yesterday: http://cran.r-project.org/web/packages/rgrass7/index.html 
Quite relevant to this discussion.

**Comment by sckott** [comment]
@jafflerbach Good idea, i'm interested. In addition, I've been working on some geo utilities pkgs recently (geojsonio, lawn, wellknown, proj, http://ropensci.org/packages/#geospatial), so I'd be interested to hear what I can do with those to support use cases you have.

**Comment by lmullen** [comment]
@jafflerbach I'd be very interested in this session.

On Wed, Mar 11, 2015 at 1:12 PM, Scott Chamberlain <notifications@github.com

**Comment by ekansa** [comment]
I'm interested in learning more too, since we're publishing lots of geospatial data (https://github.com/ekansa/open-context-py and http://opencontext.dainst.org/). It would be very useful to know what people need to do analyses with confidence.

**Comment by jamiecmontgomery** [comment]
I've been working with @jcheng5 a bit on integrating rasters into leaflet and it'd be great to continue the spatial data + R conversation with those interested today...

@sckott @lmullen @ekansa @jules32 @jread-usgs and others?

maybe after lunch?

**Comment by karthik** [comment]
I'm in too.

**Comment by jhollist** [comment]
I will follow along from afar.  The raster s in leaflet is a great idea!
Interested in hearing more.

**Comment by sckott** [comment]
i'm in, thanks @jafflerbach

**Comment by jordansread** [comment]
ready whenever

**Comment by jamiecmontgomery** [comment]
congregating in the back of the room now for those interested

**Comment by jhollist** [comment]
I am really interested in the results of this conversation.  If there are any notes taken, I would love to see them after the fact.

**Comment by timelyportfolio** [comment]
+1

**Comment by jordansread** [comment]
here is the geo processing engine I was describing: https://github.com/USGS-CIDA/geo-data-portal
@jcheng5 and a front-end: http://cida.usgs.gov/gdp/

**Comment by jordansread** [comment]
Joe had some cool stuff w/ shiny+leaflet+raster+geojson

**Comment by jordansread** [comment]
Link to @jcheng5 - https://github.com/rstudio/leaflet/tree/raster

**Comment by jcheng5** [comment]
Example for rstudio/leaflet raster branch: https://gist.github.com/jcheng5/a1c48a18d0ea3623c0e6

**Comment by jhollist** [comment]
Thanks for the links @jread-usgs and @jcheng5

**Comment by sckott** [comment]
@jafflerbach @jcheng5 where is this at now?  I assume the thing to follow is https://github.com/jcheng5/rasterfaster ?

**Comment by jcheng5** [comment]
Yes. I'm just now turning my attention back to leaflet after focusing on a different project the last few weeks.

**Comment by sckott** [comment]
Great, thanks!
